python3-dnf-plugin-local-mirror-detection.spec

```diff
@@ -0,0 +1,46 @@
+Name:       python3-dnf-plugin-local-mirror-detection
+Version:    1.0.0
+Release:    3%{?dist}
+Summary:    A DNF plugin to detect and use local mirrors if available.
+License:    GPL-3.0-only
```

Comment:
The license is GPL-3.0-or-later actually.

Reply:
Thanks, fixed!